run-dljc.sh

```diff
@@ -13,7 +13,7 @@
 # - Ensure that your CHECKERFRAMEWORK variable points to a built copy of the Checker Framework
 # - Other dependencies: perl, python2.7, awk, git, mvn
```

Comment:
When I originally wrote this script I used `cut` here, but ran into problems because of the possibility of different kinds of whitespace in the input file. `awk` protects against that - `cut` can't be used to separate a line that's divided by arbitrary whitespace, which is what `awk` is doing here.

`awk` is also extremely common, so I'm not bothered by depending on it, and the performance difference is insignificant.